pom.xml

```diff
@@ -43,6 +59,43 @@
                 </configuration>
 
             </plugin>
+
+            <!--            <plugin>-->
```

Comment:
remember about comment from previous hw

Reply:
already pushed changes from the previous task